kube_downscaler/scaler.py

```diff
@@ -46,11 +48,24 @@ def autoscale_resource(resource: pykube.objects.NamespacedAPIObject,
             logger.debug('%s %s/%s was excluded', resource.kind, resource.namespace, resource.name)
         else:
             replicas = resource.replicas
+            ignore = False
 
             if forced_uptime or (exclude and original_replicas):
                 uptime = "forced"
                 downtime = "ignored"
                 is_uptime = True
+            elif upscale_period != 'never' or downscale_period != 'never':
+                uptime = resource.annotations.get(UPSCALE_PERIOD_ANNOTATION, upscale_period)
```

Comment:
maybe use a different variable name (as `uptime` is already used in the other if branch)

Reply:
I used this variable, because of logging later